scripts/utility-vm/user-data.yaml

```diff
@@ -0,0 +1,11 @@
+#cloud-config
+# vim:syntax=yaml
```

Comment:
Can these comments be removed?

Reply:
I removed the vim line. I think the other one is shebang and should stay.